UI/AppKit/Application/ApplicationDelegate.mm

```diff
@@ -662,6 +667,9 @@ - (NSMenuItem*)createDebugMenu
     [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Clear Cache"
                                                 action:@selector(clearCache:)
                                          keyEquivalent:@""]];
+    [submenu addItem:[[NSMenuItem alloc] initWithTitle:@"Clear all Cookies"
```

Comment:
"all" should be capitalized here, per the [human interface guidelines](<https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/HumanInterfaceGuidelines/Text.md#capitalization>).